templates/english/topic-page.html

```diff
@@ -0,0 +1,474 @@
+              >
+                <gcds-link
+                  href="#"
+                  class="font-text"
+                >
+                  Heading link
+                </gcds-link>
+              </gcds-heading>
+              <gcds-text>
+                Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.
+              </gcds-text>
+            </div>
+            <div>
+              <gcds-heading
+                margin-bottom="100"
+                margin-top="0"
+                tag="h3"
+              >
+                <gcds-link
+                  href="#"
+                  class="font-text"
+                >
+                  Heading link
+                </gcds-link>
+              </gcds-heading>
+              <gcds-text>
+                Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.
+              </gcds-text>
+            </div>
+            <div>
+              <gcds-heading
+                margin-bottom="100"
+                margin-top="0"
+                tag="h3"
+              >
+                <gcds-link
+                  href="#"
+                  class="font-text"
+                >
+                  Heading link
+                </gcds-link>
+              </gcds-heading>
+              <gcds-text>
+                Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.
+              </gcds-text>
+            </div>
+            <div>
+              <gcds-heading
+                margin-bottom="100"
+                margin-top="0"
+                tag="h3"
+              >
+                <gcds-link
+                  href="#"
+                  class="font-text"
+                >
+                  Heading link
+                </gcds-link>
+              </gcds-heading>
+              <gcds-text>
+                Use action verbs, or simply list keywords to summarize the information or tasks that can be accomplished on the page it links to.
+              </gcds-text>
+            </div>
+          </gcds-grid>
+        </section>
+
+        <div>
+          <gcds-grid
+            tag="div"
+            columns-desktop="2fr 1fr"
+            columns-tablet="1fr"
+            class="mt-400"
+          >
+            <!---------- Featured ---------->
+            <!-- TODO: Update the heading, image and text content. -->
+            <section>
+              <gcds-sr-only tag="h2">
+                Featured
+              </gcds-sr-only>
+              <gcds-grid
+                tag="div"
+                columns-desktop="1fr 1fr"
+                columns-tablet="1fr 1fr"
+              >
+                <!-- TODO: Update the Featured image -->
+                <img src="https://dummyimage.com/360x203/000000/FFFFFF.png" alt="" class="img-full">
```

Comment:
Debating this in my mind, would it be helpful to have alt text here that says something like "Provide descriptive alt-text for the accompanying image"? We can go either way.

Reply:
I like having this guidance but I think it might be better to add it as a comment above. I fear if it was in the `alt` attribute, some people would end up leaving it there and only replace the image source.